atomic.d/openscap

```diff
@@ -1,6 +1,6 @@
 type: scanner
 scanner_name: openscap
-image_name: openscap
+image_name: registry.access.redhat.com/rhel7/openscap
 default_scan: cve
```

Comment:
I don't think you should have the registry name in here.  Just the rhel7/openscap.  That way people could use it with internal registries.